codegen-sbt/src/sbt-test/compiletime-codegen/test-compile/README.md

```diff
@@ -1,3 +1,14 @@
 # Test doc
 
-This test project has been copied from: https://github.com/guizmaii/poc_compile_time_caliban_client_generation
\ No newline at end of file
+This test project has been copied from:
+https://github.com/guizmaii/poc_compile_time_caliban_client_generation
+
+### Running locally
+You can run these tests using following sbt commandos:
+
+```sbt
+project codegenSbt
+++2.12.20
```

Comment:
Perhaps change this to `+2.12` instead so that we don't have to update it on scala version updates